zerver/tests/test_message_fetch.py

```diff
@@ -4988,15 +4988,15 @@ def update_message(self, msg: Message, content: str) -> None:
         rendering_result = render_message_markdown(msg, content)
         mention_backend = MentionBackend(realm_id)
         mention_data = MentionData(mention_backend, content, msg.sender)
+        message_edit_request = get_message_edit_request_object(
+            msg, hamlet, "change_one", None, None, content
```

Comment:
I'd prefer to pass all the parameter as mandatory kwargs, not having any positional parameters, for `get_message_edit_request_object`; it'd be a lot more readable.